instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/logs/LoggerE2ETests.kt

```diff
@@ -460,12 +476,26 @@ class LoggerE2ETests {
     @Test
     fun logs_logger_add_tag() {
         val testMethodName = "logs_logger_add_tag"
+        val tagValue = forge.aStringMatching("customTag[a-z0-9_:./-]{1,20}")
```

Comment:
I don't know how it affects performance, but in the recent change https://github.com/DataDog/dd-sdk-android/pull/698/files#diff-5a71e55cb00745521b8ca794d4d35ef852cd45d987fe4b56d45ea5e94e3408f1 constant part was extracted from `aStringMatching` argument. Maybe need to do the same here for the consistency?